Fix hardcode of gcc

Can not use the hardcode of gcc in cross compiling.

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>

Upstream-Status: Pending
---
 blitz/CMakeLists.txt |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/blitz/CMakeLists.txt b/blitz/CMakeLists.txt
index b6ced2d..bf2e92b 100644
--- a/blitz/CMakeLists.txt
+++ b/blitz/CMakeLists.txt
@@ -33,7 +33,7 @@ if(HAVE_MMX)
     if(NOT APPLE AND NOT WIN32)
         set( blitz_LIB_SRCS ${blitz_LIB_SRCS} asm_scale.S )
         add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/asm_scale.o
-            COMMAND gcc -c ${CMAKE_CURRENT_SOURCE_DIR}/asm_scale.S
+            COMMAND ${CMAKE_C_COMPILER} -c ${CMAKE_CURRENT_SOURCE_DIR}/asm_scale.S
             DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/asm_scale.S )
         set( blitz_LIB_EXTRA_SRCS ${CMAKE_CURRENT_BINARY_DIR}/asm_scale.o )
         set(HAVE_EXTERNAL_ASM TRUE)
-- 
1.7.3.4

